### ISSUE TITLE
[Package]: mkisofs

### ISSUE THREAD

**Issue by RazyRabbit** [termux/termux-packages#7465]
### Package description

mkisofs is a pre-mastering program to generate filesystems following the ISO-9660 specification. It supports the Joliet, Rock Ridge and Apple extensions, as well as the creation of HFS/ISO-9660 and UDF/ISO-9660 hybrid filesystem images (images that can be mounted with either filesystem driver).

This package must be added to enable ISO creation in Termux

### Home page URL

http://cdrtools.sourceforge.net

### Source code URL

https://github.com/jobermayr/cdrtools/tree/master/mkisofs

### Packaging policy acknowledgement

- [X] I have read and understand the [Packaging Policy](https://github.com/termux/termux-packages/blob/master/CONTRIBUTING.md#packaging-policy).

### Additional information

Error:

checking whether the C compiler (aarch64-linux-clang  ) works... no
configure: error: installation or configuration problem: C compiler aarch64-linux-clang not found.

**Comment by xeffyr** [comment]
> This package must be added to enable ISO creation in Termux

Have you tried `xorriso`?

Or `libisoburn` from unstable-repo?

**Comment by RazyRabbit** [comment]
it's a dependency of a project I'm compiling.

Btw, i will try to patch with xorriso in the makefile

**Comment by robertkirkman** [comment]
Hi do you still need this? I've been thinking about trying it.

**Comment by mguinhos** [comment]
Not really. It was because a project that i was trying to compile needed it, but i fixed by editing the build scripts.

**Comment by robertkirkman** [comment]
This is now available in tur-on-device.

This issue can remain open until there is some way to cross-compile it or otherwise move it to github.com/termux/termux-packages.
